### ISSUE TITLE
Improve emoji chart depiction of added sequences

### ISSUE THREAD

**Issue by nedley** [unicode-org/unicodetools#96]
Rather than using what looks like a heavy plus sign, perhaps color the entire table cell?

**Comment by macchiati** [comment]
Sounds reasonable to me (it's nice to have a character also as a stand-in,
because you can search on page for it — but with coloring it doesn't have
to be so 'in your face').

Mark

**Comment by nedley** [comment]
Good point about searching, but the consensus from Properties & Algorithms was that using any emoji is confusing. Perhaps a plain text “N/A” instead (still with coloring)?

**Comment by macchiati** [comment]
Sounds reasonable.

Mark
